main.qml

```diff
@@ -1891,6 +1954,14 @@ ApplicationWindow {
         passwordDialog.open();
     }
 
+    function changeWalletMode(mode){
```

Comment:
Suggestion: Perhaps in the Layout/UI settings tab, allow users to change wallet modes.

Reply:
Nvm. I see the mode selection button on the welcome to monero screen. Nice :)

Reply:
Still a good suggestion. I think we can think about this at a later point. Would be nice to be able to more easily switch modes (whilst you have a wallet opened).